### PR TITLE
Feature/progress policy length component

### PR DESCRIPTION
**Pull request by hdfedscottquested**

### CODE REVIEW

File: components/atoms/icon/__snapshots__/Icon.test.js.snap [hunk 1]
```diff
@@ -7,8 +7,9 @@ exports[`Check the icon renders it should match the snapshot 1`] = `
     Object {
       "backgroundColor": "#0085ff",
       "borderColor": undefined,
-      "height": 39,
-      "width": 39,
+      "borderWidth": 3,
+      "height": 60,
+      "width": 60,
```
Comment (hdfeddanjackson) [inline components/atoms/icon/__snapshots__/Icon.test.js.snap:12]: Remove the comma at the end of line 12.

File: components/atoms/policy progress/policy-progress.scss [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+	&__bar {
+		width: 100%;
+		height: 5px;
+		border-radius: 5px;
+		border: 0;
+
+		&::-webkit-progress-bar {
+			background-color: $color-brand-blue-pale-azure;
+			height: 5px;
```
Comment (hdfeddanjackson) [inline components/atoms/policy progress/policy-progress.scss:16]: The `height` and `border-radius` are common across three pseudo class selectors, could we look to share these using a single piece of code?

File: components/atoms/policy progress/policy-progress.scss [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+			background-color: $color-brand-seaweed;
+			height: 5px;
+			border-radius: 5px;
+		}
+
+		&[value^="11"]:not([value="11"])::-webkit-progress-value {
+			background-color: $color-brand-crab;
+		}
+
+		&[value^="11"]:not([value="11"])::-moz-progress-bar{
```
Comment (hdfeddanjackson) [inline components/atoms/policy progress/policy-progress.scss:36]: Can this be combined with the code on line 32 as the `background-color` is the same?

File: components/atoms/policy progress/policy-progress.scss [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+	&--dark {
+		color: $color-white;
+	}
+
+	&--error {
+
+		.a-policy-progress__bar {
+
+			&::-webkit-progress-value {
```
Comment (hdfeddanjackson) [inline components/atoms/policy progress/policy-progress.scss:63]: I think every one of these selectors can be combined as you're repeating the same property for all four.

Comment (hdfedscottquested) [reply components/atoms/policy progress/policy-progress.scss:63]: As my comment above - chaining these selectors doesn't work

File: src/scss/variables/variables.scss [hunk 5]
```diff
@@ -12,13 +12,16 @@ $color-brand-blue-mid-dark: #00448c;
 $color-brand-blue-grey-dark: #012245;
```
Comment (hdfeddanjackson) [inline src/scss/variables/variables.scss:12]: Should we be using the `!default` keyword for our colour variables?

Comment (hdfedscottquested) [reply src/scss/variables/variables.scss:12]: Probably not. We're not going to be reassigning the variables (like you would in a theme or template) so I'm not sure they're needed.

**Review by hdfeddanjackson — APPROVED** [review]
Approved.

**Pull request merged**